conf/master

```diff
@@ -297,6 +297,11 @@
 #batch_safe_limit: 100
 #batch_safe_size: 8
 
+# Master stats enables stats events to be fired from the master at close
+# to the defined interval
+#master_stats: False
```

Comment:
That's a very nice feature. Can we also document these options? https://docs.saltstack.com/en/latest/ref/configuration/master.html Thanks!